### PR TITLE
fix(UI): Added code to show the project list in component tab

### PR DESCRIPTION
**Pull request by nikkuma7**
Signed-off-by: Nikesh kumar <kumar.nikesh@simens.com>

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
> * Which issue is this pull request belonging to and how is it solving it? #1697 
> * Did you add or update any new dependencies that are required for your change? No

Issue: 
There is an issue with project list which is not showing in component tab.

### How To Test?
1: Login sw360 
2: Go to component tab.
3: select any component and go inside the component.
4:scroll down and see the project list.

> How should these changes be tested by the reviewer?
> Have you implemented any additional tests?

### Checklist
Must:
- [ ] All related issues are referenced in commit messages and in PR

### CODE REVIEW

File: frontend/sw360-portlet/src/main/resources/META-INF/resources/html/components/includes/components/detailOverview.jspf [hunk 1]
```diff
@@ -102,6 +102,7 @@
 		                    <%@include file="/html/components/includes/components/summary.jspf" %>
 
                             <core_rt:set var="documentName"><sw360:out value='${component.name}'/></core_rt:set>
+                             <core_rt:set var="tableId" value="usingProjectsTableSummary"/>
```
Comment (akapti) [inline frontend/sw360-portlet/src/main/resources/META-INF/resources/html/components/includes/components/detailOverview.jspf:105]: align it properly.

**Review by akapti — CHANGES_REQUESTED** [review]
tested the PR, working fine.
screenshot of component details page:
image



The exact same issue is present for Projects details page as well, where in sub projects are not listing it's parents projects(used by). attached sample screenshot below. Please fix it as well.

image

File: frontend/sw360-portlet/src/main/resources/META-INF/resources/html/components/includes/components/detailOverview.jspf (hunk 1 above)
Comment (nikkuma7) [reply frontend/sw360-portlet/src/main/resources/META-INF/resources/html/components/includes/components/detailOverview.jspf]: done.

**Review by akapti — APPROVED** [review]
Code looks good.
tested the changes and used by projects are listing in both project and component details page.

Testing screenshot:
image

**Pull request merged**